android/gradle.properties

```diff
@@ -16,5 +16,3 @@ org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryErro
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
-
-android.useDeprecatedNdk=true
```

Comment:
hm?

Reply:
while building example project i noticed:

![image](https://user-images.githubusercontent.com/18573330/61053140-3dbec580-a3ed-11e9-83e8-109b36cec6c1.png)

so decided to remove it